pkgs/development/python-modules/sortedl1/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pybind11,
+  scikit-build-core,
+  numpy,
+  scikit-learn,
+  scipy,
+  cmake,
+  ninja,
+}:
+
+buildPythonPackage rec {
+  pname = "sortedl1";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
fetchFromGitHub is preferred here unless there is a compelling reason to use Pypi

Reply:
I don't mind changing, but why is `fetchFromGitHub` preferred? I cannot see anything about this in the docs (https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md).